### PR TITLE
Use arrow keys to select buffer in buffer search

### PR DESCRIPTION
**Pull request by Bakke**

### CODE REVIEW

File: js/glowingbear.js [hunk 1]
```diff
@@ -750,17 +750,39 @@ weechat.controller('WeechatCtrl', ['$rootScope', '$scope', '$store', '$timeout',
...
             $scope.search = '';
         } // Handle enter
         else if (code === 13) {
+            var index;
             $event.preventDefault();
             if ($scope.filteredBuffers.length > 0) {
-                $scope.setActiveBuffer($scope.filteredBuffers[0].id);
+                // Go to highlighted buffer if available
+                // or first one
+                if( $scope.search_highlight_key ) {
```
Comment (torhve) [inline js/glowingbear.js]: Whitespace should be fixed here.

File: js/glowingbear.js [hunk 2]
```diff
@@ -750,17 +750,39 @@ weechat.controller('WeechatCtrl', ['$rootScope', '$scope', '$store', '$timeout',
...
             }
             $scope.search = '';
+        } // Handle arrow up
+        else if (code === 38) {
+            $event.preventDefault();
+            if ($scope.search_highlight_key && $scope.search_highlight_key > 0) {
+                $scope.search_highlight_key = $scope.search_highlight_key - 1;
+            }
+        } // Handle arrow down and tab
+        else if(code === 40 || code === 9) {
```
Comment (torhve) [inline js/glowingbear.js]: Whitespace

**Comment by torhve** [comment]
I like it. Very neat.

File: js/glowingbear.js [hunk 3]
```diff
@@ -750,17 +750,39 @@ weechat.controller('WeechatCtrl', ['$rootScope', '$scope', '$store', '$timeout',
...
             $scope.search = '';
         } // Handle enter
         else if (code === 13) {
+            var index;
             $event.preventDefault();
             if ($scope.filteredBuffers.length > 0) {
-                $scope.setActiveBuffer($scope.filteredBuffers[0].id);
+                // Go to highlighted buffer if available
+                // or first one
+                if($scope.search_highlight_key) {
```
Comment (torhve) [inline js/glowingbear.js]: Whitespace :-)

**Comment by lorenzhs** [comment]
awesome, thanks!

**Pull request merged**